.github/workflows/ci-data.yml

```
@@ -42,7 +42,7 @@ jobs:
 
       - uses: google-github-actions/auth@v2
         with:
-          credentials_json: ${{ secrets.GCP_CREDENTIALS }}
+          workload_identity_provider: "${{ secrets.WIF_PROVIDER_NAME }}"
```

Per https://github.com/google-github-actions/auth/blob/main/README.md#preferred-direct-workload-identity-federation we probably need a `project_id` here, too. With key files, it can pull that from the JSON, but I don't think the default project ID is available in most other authentication methods.

```suggestion
          project_id: 'ibis-gbq'
          workload_identity_provider: "${{ secrets.WIF_PROVIDER_NAME }}"
```

Also, I got approval and created the provider

```
gcloud iam workload-identity-pools providers describe "ibis" \
  --project="${PROJECT_ID}" \
  --location="global" \
  --workload-identity-pool="github" \
  --format="value(name)"
```

Output:

```
projects/546535678771/locations/global/workloadIdentityPools/github/providers/ibis
```

FYI for making the "secret". Alternatively, I think it'd be fine to hardcode this.

---

Hardcoding it seems a bit more understandable, so I'll do that.

---

Actually an environment variable maybe is slightly better only because we repeat this in 3 places?